pkg/asset/cluster/tfvars.go

```diff
@@ -79,7 +79,7 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 	masters := mastersAsset.Machines()
 	masterCount := len(masters)
 	data, err := tfvars.TFVars(
-		clusterID.ClusterID,
+		clusterID.HumanID,
 		installConfig.Config.ObjectMeta.Name,
```

Comment:
Can we drop this?  A non-uniquified name seems like something we'd never use.

Reply:
https://github.com/openshift/installer/pull/1280/commits/3711b75d6bf53e2c7bfd0e15af7d5044b06cebbf